sympy/stats/crv_types.py

```diff
@@ -551,7 +551,8 @@ def BetaNoncentral(name, alpha, beta, lamda):
     /____,
     k = 0
 
-    Compute cdf with specific 'x', 'alpha', 'beta' and 'lamda' values as follows :
+    Compute cdf with specific 'x', 'alpha', 'beta' and 'lamda' values as follows:
```

Comment:
I think its about new line addition and a space removal, right?

Reply:
@czgdp1807 That's Right : )

Reply:
> I don't know why the workflow for checks are not started

This is standard for first-time contributors. Now it is started.

Reply:
Oh, yeah. That is since you seem to be logged in to a different account. The commit is from @KuldeepBorkar and your PR is opened from @Kuldeep-GitWorks 

I do not know the background to multiple accounts, but consider what @anutosh491 says above. It looks a bit weird reacting on your own comments.

Reply:
@oscargus Yeah I ran into a problem for which I have written a complete mail on sympy.
The actual reason for reacting to comment, I forgot ; )
Maybe due to using multiple accounts it maybe a mistake I will remove the reaction now.